examples/nextjs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "jest --watch",
+    "test:ci": "jest --ci"
+  },
```

Comment:
Shall we include the `"create-app-definition"` `"upload"` and `"upload-ci"` as we have in the other templates?

Reply:
Won't work here. This is SSR and can't be hosted statically

Reply:
True, but the `create-app-definition` will work or not?

Reply:
yes, added that